### PR TITLE
[Author] Update EaselJS package.json w/ npm auto-update config

### PR DESCRIPTION
**Pull request by wdamien**
Added npmFileMap for auto update support.

Pull request for issue: #11362
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/CreateJS/Easeljs
 * Official website (optional, not the repository): http://createjs.com/easeljs
 * NPM package url (optional): https://www.npmjs.com/package/easeljs
 * License and its reference: MIT - https://github.com/CreateJS/EaselJS/blob/master/LICENSE.txt
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 409
   - Count of stars: 6.336
   - Count of forks: 1704
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [ ] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [ ] The parent of the commit(s) in the PR is not old than 3 days.
 * [ ] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by sufuf3** [comment]
Helped rebase master and squash the commit.

@wdamien May I ask that which auto-update do you like to use, npm or git?
Because I found that there are 3 releases versions from npm, and 17 releases versions from GitHub.
As usual, we prefer to use more releases versions as the auto-update option.
But if you think npm is stable than GitHub repo, then npm is a good choice.

@PeterDaveHello This PR is duplicated with #11363, I asked the contributor of #11363 but there is no response 14 days ago. And this PR sent by the lib's author.
What can we do for this situation?

Thank you.

**Comment by PeterDaveHello** [comment]
@sufuf3 so I think we can take this one once everything is good here?

**Comment by sufuf3** [comment]
OK! Got it. 
Thank you.

**Comment by PeterDaveHello** [comment]
@sufuf3 You're welcome.

File: ajax/libs/EaselJS/package.json [hunk 1]
```diff
@@ -2,23 +2,28 @@
...
-    "mikechambers",
-    "wdamien"
+  "author": "gskinner.com, inc.",
+  "homepage": "https://createjs.com/easeljs/",
+  "npmName": "easeljs",
+  "npmFileMap": [
+    {
+      "basePath": "lib",
+      "files": [
+        "**/*"
```
Comment (PeterDaveHello) [inline ajax/libs/EaselJS/package.json]: Shouldn't we exclude `README.md` here?

**Comment by wdamien** [comment]
@sufuf3 Is the above CI error something I need to update? It seems like when you merge this PR the rebase should be fine? (Same with https://github.com/cdnjs/cdnjs/pull/11854, https://github.com/cdnjs/cdnjs/pull/11853 and https://github.com/cdnjs/cdnjs/pull/11852)

**Comment by PeterDaveHello** [comment]
@wdamien I'll help squash the commits into only one commit and rebase the branch with the latest master branch.

**Comment by PeterDaveHello** [comment]
cc #9145

**Pull request merged**